docs/src/modules/developing/pages/development-process-js.adoc

```diff
@@ -57,17 +57,16 @@ For more details and examples take a look at the following topics:
 [#_create_unit_tests]
 == Create unit tests
 
-It is good practice to write unit tests as you implement your services. The JavaScript kickstart codegen project packages Mocha and Chai and is configured for test-driven development. For implementation details see the https://developer.lightbend.com/docs/akka-serverless/tutorial/node-cart-test.html#_ii_add_unit_tests[Shopping cart tutorial - Create unit tests] topic.
+It is good practice to write unit tests as you implement your services. The JavaScript kickstart codegen project packages Mocha and Chai and is configured for test-driven development.
```

Comment:
@retgits as far as I can tell, docs are missing for::

- Unit testing [Value Entities using the JavaScript SDK](https://github.com/lightbend/akkaserverless-javascript-sdk/issues/179)
- Unit testing [Event Sourced Entities using the JavaScript SDK](https://github.com/lightbend/akkaserverless-javascript-sdk/issues/178)

Both Quickstarts are also missing unit test instructions:

- https://developer.lightbend.com/docs/akka-serverless/quickstart/cr-value-entity-javascript.html
- https://developer.lightbend.com/docs/akka-serverless/quickstart/sc-eventsourced-entity-javascript.html

For all of the above reasons I think we're better off simply removing this broken link for now.

Reply:
Those two quickstart never had unit test instructions. They're intended to get a project running on Akka Serverless as fast as possible. We can create other quickstarts to showcase unit testing though.

Reply:
Slight correction. I stumbled upon [this](https://developer.lightbend.com/docs/akka-serverless/javascript/kickstart.html#_unit_and_integration_tests). I missed it because when I searched the docs, my eyes were scanning for the word "JavaScript", not "Node".

Regardless the unit testing described here appears to be using an outdated version of the SDK, so it's not worth linking to at this time.